crates/uv/tests/it/lock.rs

```diff
@@ -19421,8 +19421,6 @@ fn lock_multiple_sources_index_disjoint_extras_with_marker() -> Result<()> {
         version = "3.1.3"
         source = { registry = "https://download.pytorch.org/whl/cu124" }
         resolution-markers = [
-            "sys_platform == 'darwin'",
-            "sys_platform != 'darwin'",
```

Comment:
Is this a bug, perhaps? Why write it when it's empty?

Reply:
I think writing it when it's empty is probably an extant bug, perhaps exacerbated by conflicting extras. (Or maybe even only made possible by it.) I can look into it.

But as for deleting the markers here, I believe it's correct. I wrote up an explanation for it in the commit message.